catalog/async/catalog-async-postingestplugin/src/test/groovy/org/codice/ddf/catalog/async/processingplugin/ProcessingPostIngestPluginTest.groovy

```diff
@@ -85,7 +85,7 @@ class ProcessingPostIngestPluginTest extends Specification {
         then:
         output == input
 
-        3 * catalogFramework.getResource(_ as ResourceRequest, sourceId) >> Mock(ResourceResponse) {
+        0 * catalogFramework.getResource(_ as ResourceRequest, sourceId) >> Mock(ResourceResponse) {
```

Comment:
The catalogFramework is no longer being called because that flow is being shoved in a supplier and passed into the lazy loaded process resource.

Make the change as suggested.

Also added a test for the stuff in getProcessResource